nextcloudappstore/scaffolding/tests/data/info.xml

```diff
@@ -16,7 +16,7 @@
     <category>tools</category>
     <bugs>https://test.com</bugs>
     <dependencies>
-        <nextcloud min-version="24" max-version="24"/>
+        <nextcloud min-version="28" max-version="28"/>
```

Comment:
Shouldn't the min version be 27 because the template is also for 27?

Reply:
tests were modified to test against 28 version, to not modify them in 3 months again..